apps/convergecfd/run_ccfd_2.4_impi.pbs

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+#PBS -N ConvergeCFD
+#PBS -l select=1:ncpus=120:mpiprocs=120:mem=440gb:pool_name=compute2
+#PBS -joe
+
+APPLICATION=convergecfd
+APP_INSTALL_DIR=${APP_INSTALL_DIR:-/apps}
+CCFD_VERSION=${CCFD_VERSION:-2.4.23}
+EX_PATH=${EX_PATH:-${APP_INSTALL_DIR}/convergecfd/$CCFD_VERSION/example_cases/Internal_Combustion_Engines/Gasoline_spark_ignition_PFI}
+CASE=${CASE:-SI8_engine_premix_SAGE}
+MPI=intel
+LICENSE_INFO=${LICENSE_INFO:-2765@10.1.0.5}
+
+DATE=`date "+%Y%m%d-%H%M%S-%4N"`
+CORES=$(cat $PBS_NODEFILE | wc -l)
+NODES=$(cat $PBS_NODEFILE | sort -u | wc -l)
+
+export RLM_LICENSE=$LICENSE_INFO
+export PATH=${APP_INSTALL_DIR}/convergecfd/$CCFD_VERSION/l_x86_64/bin:$PATH
+
+# Setup Intel MPI
+module load mpi/impi-2019
+source $MPI_BIN/mpivars.sh -ofi_internal
+export I_MPI_FABRICS="shm:ofi"
+export I_MPI_FALLBACK_DEVICE=0
```

Comment:
I_MPI_FALLBACK_DEVICE can be removed when using OFI as it is no longer supported

Reply:
I have removed I_MPI_FALLBACK_DEVICE from the script